removing .todos.json from a synced list will cause crash

synced lists cannot handle when a synced list's `.todos.json` is removed.  

To repro:

1. Sync a list with ultralist.io
2. `rm -rf .todos.json`
3. Run `ultralist init` 
4. See crash.

```
Repo initialized.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x72a670]

goroutine 1 [running]:
github.com/gammons/ultralist/ultralist.(*EventLogger).LoadSyncedLists(0xc420074300)
        /home/grant/code/go/src/github.com/gammons/ultralist/ultralist/event_logger.go:129 +0x1a0
github.com/gammons/ultralist/ultralist.(*App).InitializeRepo(0xc420095050)
        /home/grant/code/go/src/github.com/gammons/ultralist/ultralist/app.go:36 +0xf9
main.routeInput(0x7fffcf7071b3, 0x4, 0x7fffcf7071b3, 0x4)
        /home/grant/code/go/src/github.com/ultralist/ultralist/main.go:58 +0x791
main.main()
        /home/grant/code/go/src/github.com/ultralist/ultralist/main.go:21 +0xcf
```

`LoadSycnedLists` should have a grooming step to remove lists that no longer exist.

--- a/ultralist/app.go
+++ b/ultralist/app.go
@@ -57,7 +57,7 @@ func (a *App) InitializeRepo() {
 	fmt.Println("Repo initialized.")
 
 	backend := NewBackend()
-	eventLogger := &EventLogger{Store: a.TodoStore}
+	eventLogger := &EventLogger{Store: a.TodoStore, CurrentTodoList: a.TodoList}
 	eventLogger.LoadSyncedLists()
 
 	if !backend.CredsFileExists() {

